Exec/RegTests/FlameSheet/GNUmakefile

```diff
@@ -23,12 +23,12 @@ DEFINES+=-DALWAYS_NEW_J
 DO_2S_CONVERGENCE=FALSE
 
 #######################
-USE_SUNDIALS_PP = FALSE
+USE_SUNDIALS_PP = TRUE
```

Comment:
do we want that by default for the regression test case?

Reply:
Ah crap it's going to break the regtests for sure... Would be nice to use cvode though. Marc ?

Reply:
Hey, I didn't set that up.  We can add another reg test that sets it to TRUE though

Reply:
Ok I'll revert